custom_components/tuya_local/devices/vacplus_dehumidifier.yaml

```diff
@@ -0,0 +1,145 @@
+name: Vacplus Dehumidifier
+products:
+  - id: yttlroi7nr18vqbo
+    name: Vacplus Dehumidifier
+primary_entity:
+  entity: humidifier
+  class: dehumidifier
+  dps:
+    - id: 1
+      name: switch
+      type: boolean
+      mapping:
+        - dps_val: false
+          icon: "mdi:air-humidifier-off"
+          icon_priority: 1
+        - dps_val: true
+          icon: "mdi:air-humidifier"
+          icon_priority: 4
```

Comment:
Remove these priorities, as the icon is not set elsewhere.  Actually you can remove the whole mapping, because these icons are basically the default, but setting them explicitly overrides Home Assistant theming.